### DISCUSSION TITLE
【1136380】终极钓鱼模拟器2/Ultimate Fishing Simulator 2

### DISCUSSION THREAD

**Discussion by qqa464684039** [pjy612/SteamManifestCache#353] [Share]
appid 1136380
```
1136381;f6ab2eb01876bb0a71ed82d311cff335764127e35b53358cbf95733bacb1d2f6
```
[1136380.zip](https://github.com/pjy612/SteamManifestCache/files/13883368/1136380.zip)

**Comment by pjy612** [comment]
请修正密钥格式 #26

**Reply by qqa464684039** [reply]
1136381:f6ab2eb01876bb0a71ed82d311cff335764127e35b53358cbf95733bacb1d2f6

**Reply by pjy612** [reply]
请编辑原帖，以前分享了就分享了。
现在需要正规一点，至少 统一规范
